plugins/hydra_ax/setup.py

```diff
@@ -22,7 +22,7 @@
             "Programming Language :: Python :: 3.7",
             "Operating System :: OS Independent",
         ],
-        install_requires=["hydra-core"],
+        install_requires=["hydra-core", "ax-platform==0.1.6", "SQLAlchemy==1.3.11"],
```

Comment:
Can you talk to the Ax team and see what are the proper dependencies they recommend?